opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/SpanFinishingFilter.java

```diff
@@ -27,18 +27,8 @@
  */
 public class SpanFinishingFilter implements Filter {
 
-  private final Tracer tracer;
-
-  public SpanFinishingFilter() {
-    this(GlobalTracer.get());
-  }
-
-  public SpanFinishingFilter(Tracer tracer){
```

Comment:
To keep backwards compatibility, could you keep the constructor, even if it's noop? Bonus points if you add a `@Deprecated` annotation and javadoc with the reasoning. I know that, according to semver, we are still allowed to have breaking changes, but we aware of places using this constructor (SmallRye/WildFly are the first ones that come to mind).

Reply:
I am fine with breaking this as it is under 0.x. But you are right it's better to deprecate first.